tests/unit_tests/test_variants.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from UnleashClient.variants import Variants
+from tests.utilities.mocks.mock_variants import VARIANTS
+
+
+@pytest.fixture()
+def variations():
+    yield Variants(VARIANTS, "TestFeature")
+
+
+def test_variations_override_match(variations):
+    override_variant = variations._apply_overrides({'userId': 'ivanklee86@gmail.com'})
```

Comment:
not sure if you want to expose your email address to the world?

Reply:
Oops. >_> <_<